catalog/core/catalog-core-solr/pom.xml

```diff
@@ -128,7 +123,7 @@
                             <include>ddf.platform.solr:solr-xpath</include>
                             <include>org.apache.lucene:lucene-core</include>
                             <include>ddf.catalog.core:catalog-core-api-impl</include>
-                            <include>com.spatial4j:spatial4j</include>
+                            <include>org.locationtech.spatial4j:spatial4j</include>
```

Comment:
It figures it out from the Solr dependencies.  This should make it easier to do Solr updates in the future. You can see the same thing in the Solr dependencies project under platform/solr.